nevergrad/optimization/lamcts/MCTS.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+# verify
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import json
+import collections
+import copy as cp
+import math
+from collections import OrderedDict
+import os.path
+import numpy as np
+import time
+import operator
+import sys
+import pickle
+import os
+import random
+from datetime import datetime
+from .Node import Node
+from .Net_Trainer import Net_Trainer
+from .utils import latin_hypercube, from_unit_cube
+from .functions import *
+from torch.quasirandom import SobolEngine
+import torch
+#from functions.rover_function import rover_nag
+#from functions.mujoco_functions import *
+# from push_function import PushReward
+
+class MCTS:
+    #############################################
+
+    def __init__(self, lb, ub, dims, ninits, func, device):
+        self.dims                    =  dims
+        self.samples                 =  []
+        self.nodes                   =  []
+        self.Cp                      =  5
```

Comment:
Please change the Cp to max(function value)* 0.1 for every test functions. The problem of MCTS is that one Cp does not fit for all.